modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1107,43 +1107,93 @@ Command::
 +
 [source, cypher]
 ----
-REVOKE GRANT WRITE ON GRAPH * FROM Foo
+REVOKE GRANT WRITE ON GRAPH * FROM manager
 ----
 
 Full title of the returned code::
-`REVOKE GRANT WRITE ON GRAPH * FROM Foo` has no effect.
+`REVOKE GRANT WRITE ON GRAPH * FROM manager` has no effect.
 
 Description of the returned code::
 Role does not exist. Make sure nothing is misspelled.
 This notification will become an error in a future major version.
 See Status Codes documentation for more information.
 
 Suggestions for improvement::
-Make sure nothing is misspelled and that this was the intended role.
+Verify that this is the intended role and that it was not misspelled.
 ====
 
-.Revoking a privilege on a non-existing graph/database from a role
+.Revoking a privilege on a non-existing graph from a role
 ====
 Command::
 +
 [source, cypher]
 ----
-REVOKE GRANT WRITE ON GRAPH Foo FROM editor
+REVOKE GRANT WRITE ON GRAPH neo3j FROM editor
 ----
 
 Full title of the returned code::
-`REVOKE GRANT WRITE ON GRAPH Foo FROM editor` has no effect.
+`REVOKE GRANT WRITE ON GRAPH neo3j FROM editor` has no effect.
 
 Description of the returned code::
-Database 'Foo' does not exist. Make sure nothing is misspelled.
+Graph 'neo3j' does not exist. Make sure nothing is misspelled.
```

Comment:
I don't think this one would change either. The current error for grant/deny says: `Database 'X' does not exist.` for both graph and database